webpack.config.js

```diff
@@ -103,6 +103,7 @@ const config = {
 			minChunks: (module) => module.context && module.context.indexOf("node_modules") !== -1,
 		}),
 	],
+	stats: "minimal",
```

Comment:
This effectively removes all information from the Webpack output:

```
$ npm run build
> webpack
   332 modules
```

I know you have #1942 that improves output of `npm run start-dev` but it doesn't fix it for `npm run build` and `npm run watch`. I would not include this change.